src/realm/util/file.hpp

```diff
@@ -649,7 +649,7 @@ class File {
         size_t m_reservation_size = 0;
         size_t m_offset = 0;
         FileDesc m_fd;
-        AccessMode m_a;
+        AccessMode m_access_mode = access_ReadOnly;
```

Comment:
Copying a default-initialized MapBase was UB because it assigned an initialized value to an enum. i also renamed the field because I had no idea what `m_a` was.